.eslintrc.json

```diff
@@ -45,7 +48,7 @@
     "jsx-a11y/anchor-is-valid": "off",
     "import/no-unresolved": "off",
     "import/extensions": "off",
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
+    "import/no-extraneous-dependencies": ["off", { "devDependencies": true }],
```

Comment:
**suggestion:** since this is off don't make sense to still have it as a devDependencies

```suggestion
    "import/no-extraneous-dependencies": "off",
```